website/docs/r/oracledatabase_exadata_infrastructure.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Oracle Database"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_oracle_exadata_infrastructure"
+description: |-
+  Manages a Exadata Infrastructure.
```

Comment:
```suggestion
  Manages an Exadata Infrastructure.
```

Reply:
Changed to `Cloud Exadata Infrastructure`.